packages/sdk/python/human-protocol-sdk/human_protocol_sdk/gql/operator.py

```diff
@@ -19,8 +19,14 @@
     fee
     publicKey
     webhookUrl
-    url,
+    url
     jobTypes
+    reputationNetwork {
```

Comment:
I think it should only have reputationNetworks

Reply:
Yes, removed